mypy/build.py

```diff
@@ -117,6 +117,10 @@
     "typing_extensions",
     "mypy_extensions",
     "_importlib_modulespec",
```

Comment:
OT, but this module doesn't seem to exist

Reply:
Yeah, it turns out it was deleted in https://github.com/python/typeshed/pull/5350